lib/tokens/token.js

```diff
@@ -41,6 +41,18 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
     this.uid = details.uid || null
     this.lifetime = details.lifetime || Infinity
     this.createdAt = optionallyOverrideCreatedAt(details.createdAt)
+    if (details.createdAt) {
```

Comment:
It possible, yep, although that isn't the case I want to investigate right now. The ones that I'm puzzled about are the things that look like real timestamps. Zeroes we know we can ignore, they're never correct, but the actual timestamps are what stopped us landing a fix for the password reset token issue.